src/test/java/nextstep/subway/section/SectionAcceptanceTest.java

```diff
@@ -0,0 +1,150 @@
+package nextstep.subway.section;
+
+import static nextstep.subway.line.LineAcceptanceTestFixture.지하철_노선_생성;
+import static nextstep.subway.line.LineAcceptanceTestFixture.지하철_노선_조회;
+import static nextstep.subway.section.SectionAcceptanceTestFixture.지하철_노선_구간_추가;
+import static nextstep.subway.station.StationAcceptanceTestFixture.지하철역_생성;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.List;
+import nextstep.subway.AbstractAcceptanceTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+@DisplayName("지하철 구간 관련 기능")
+public class SectionAcceptanceTest extends AbstractAcceptanceTest {
+    
+    private Long 강남역_ID;
+    
+    private Long 광교역_ID;
+    
+    private Long 신규역_1_ID;
+    
+    private Long 신규역_2_ID;
+    
+    @BeforeEach
+    void addSection() {
+        강남역_ID = 지하철역_생성("강남역").jsonPath().getLong("id");
+        광교역_ID = 지하철역_생성("광교역").jsonPath().getLong("id");
+        신규역_1_ID = 지하철역_생성("양재역").jsonPath().getLong("id");
+        신규역_2_ID = 지하철역_생성("신사역").jsonPath().getLong("id");
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 역 사이에 새로운 역을 등록하면
+     * Then 새로운 구간이 추가된다.
+     */
+    @DisplayName("역 사이에 새로운 역을 등록한다.")
+    @Test
+    void addStationBetweenLine() {
+        // given
+        Long 신분당선_ID = 지하철_노선_생성("신분당선", "주황색", 강남역_ID, 광교역_ID, 12).jsonPath().getLong("id");
+
+        // when
+        지하철_노선_구간_추가(신분당선_ID, 강남역_ID, 신규역_1_ID, 5);
+
+        // then
+        List<String> result = 지하철_노선_조회(신분당선_ID).jsonPath().getList("stations.name");
+        assertThat(result).hasSize(3)
+            .contains("강남역", "광교역", "양재역");
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 상행 종점에 지하철 구간을 등록하면
+     * Then 노선에 새로운 지하철 역이 추가된다
+     */
+    @DisplayName("새로운 역을 상행 종점으로 등록한다.")
+    @Test
+    void addUpStationSection() {
+        // given
+        Long 신분당선_ID = 지하철_노선_생성("신분당선", "주황색", 강남역_ID, 광교역_ID, 12).jsonPath().getLong("id");
+
+        // when
+        지하철_노선_구간_추가(신분당선_ID, 신규역_1_ID, 강남역_ID, 3);
+
+        // then
+        List<String> result = 지하철_노선_조회(신분당선_ID).jsonPath().getList("stations.name");
+        assertThat(result).hasSize(3)
+            .contains("강남역", "광교역", "양재역");
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 하행 종점에 지하철 구간을 등록하면
+     * Then 노선에 새로운 지하철 역이 추가된다
+     */
+    @DisplayName("새로운 역을 하행 종점으로 등록한다.")
+    @Test
+    void addDownStationSection() {
+        // given
+        Long 신분당선_ID = 지하철_노선_생성("신분당선", "주황색", 강남역_ID, 광교역_ID, 12).jsonPath().getLong("id");
+
+        // when
+        지하철_노선_구간_추가(신분당선_ID, 광교역_ID, 신규역_1_ID, 3);
```

Comment:
개인적으로 신규역 이라는 변수명 보다, 다른 역처럼 이름으로 네이밍 하는게 어떨까 싶네요 :)
테스트를 읽어내려가면서 신규역이라는 의미는 전달이 오히려 잘 되는데, 실제 어떤 역인지 알기 위해서 선언부를 다시 찾아야하는 느낌이 있는 것 같아서요 ㅎㅎ
개인적인 생각입니다 😁
```suggestion
        지하철_노선_구간_추가(신분당선_ID, 광교역_ID, 양재역_ID, 3);
```